deepchem/models/torch_models/cgcnn.py

```diff
@@ -246,6 +250,53 @@ def forward(self, dgl_graph):
       proba = F.softmax(logits)
       return proba, logits
 
+class CGCNN_OV(CGCNN):
+  """Version of Crystal Graph Convolutional Neural Network (CGCNN) adapted for conversion to IR.
+  """
+
+  def __init__(self, in_node_dim, hidden_node_dim, in_edge_dim,
+                num_conv, predictor_hidden_feats, n_tasks, mode, n_classes
+    ):
+    super().__init__(in_node_dim, hidden_node_dim, in_edge_dim,
+                      num_conv, predictor_hidden_feats, n_tasks, mode, n_classes)
+    self.graph = None
+  
+  def forward(self, inputs):
```

Comment:
Will ONNX export work if we call parent's forward here?

Reply:
It's still actual. Can we redefine `self.pooling` so parent's forward can be used?

Reply:
No, because we still have input as list and self.graph doesn't have `x` and `edge_attr`